packages/pwa-kit-runtime/src/utils/middleware/slas-private-client.js

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
In general I think this middleware is not generic enough to live in the `runtime`. The Runtime doesn't know what SLAS is, nor should it. 

Lets think of an alternative solution for implementing this middleware that balances out the ease of use (that your middleware has for end users) with flexibility and sense of belonging in the Runtime project. 

This is what I had in mind when we had our previous discussions (this is rough): 

```
// ssr.js - In the template project
import {ProxyHeaderRewriteMiddleware} from 'runtime'

const vincentsNewMiddleware = new ProxyHeaderRewriteMiddleware({
    rewrite: [{
        path: '/ssr/auth'
        headers: {
             [`Authorization`]: `the_new_value`,
             ....
        }
    }, ...]
})

app.use(vincentsNewMiddleware)
```